contrib/scripts/kind.sh

```diff
@@ -34,12 +34,12 @@ if ! have_kind; then
     echo "  https://kind.sigs.k8s.io/docs/user/quick-start/#installation"
 fi
 
-if [[ "${#}" -gt 5 ]]; then
+if [[ "${#}" -gt 6 ]]; then
   usage
   exit 1
 fi
 
-if [[ "${#}" -gt 5 ]] ||
+if [[ "${#}" -gt 6 ]] ||
    [[ "${CONTROLPLANES}" == "-h" ||
       "${CONTROLPLANES}" == "--help" ]]; then
```

Comment:
First condition can never evaluate to `true` due to the previous check line 37 right?

```suggestion
if [[ "${CONTROLPLANES}" == "-h" || "${CONTROLPLANES}" == "--help" ]]; then
```

Reply:
:bulb: